sdk/storage/azure-storage-blob/tests/test_append_blob.py

```diff
@@ -271,7 +271,7 @@ def test_append_block_from_url_with_source_if_modified(self):
         source_blob_client = self._create_source_blob(source_blob_data)
         source_blob_properties = source_blob_client.get_blob_properties()
         sas = source_blob_client.generate_shared_access_signature(
-            permission=BlobPermissions.READ + BlobPermissions.DELETE,
+            permission=BlobSasPermissions(read=True, write=True),
```

Comment:
Should be `read=True, delete=True`

Reply:
There's quite a few of these - I wont comment all of them

Reply:
Cons of carelessly mass refactoring 😬